src/sources/util/multiline_config.rs

```diff
@@ -18,6 +18,7 @@ pub struct MultilineConfig {
     #[configurable(metadata(docs::examples = "\\\\$"))]
     #[configurable(metadata(docs::examples = "^(INFO|ERROR) "))]
     #[configurable(metadata(docs::examples = ";$"))]
+    #[configurable(metadata(docs::human_name = "Timeout"))]
```

Comment:
Mistakenly copypasted entirely, or just never updated for the field at hand?

Reply:
Oops, that was a mistake.